src/components/Book.jsx

```diff
@@ -1,12 +1,48 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import Button from './Button';
+import './Book.scss';
 
 const Book = ({ book, onDelete }) => (
-  <div>
-    <h3>{book.title}</h3>
-    <p>{book.author}</p>
-    <Button onClick={onDelete}>Delete</Button>
+  <div className="book-container">
+    <div className="left-col">
+      <div className="book-details">
+        <p className="book-category">{book.category}</p>
+        <h3 className="book-title">{book.title}</h3>
```

Comment:
- [ ] Kindly use the correct font-family for the book title your font is `Montserrat`, while the given font is `RobotoSlab`